src/views/Farms/components/FarmCard/CardActionsContainer.tsx

```diff
@@ -68,15 +68,17 @@ const CardActions: React.FC<FarmCardActionsProps> = ({ farm, account, addLiquidi
     )
   }
 
+  const tokenEarnedString = TranslateString(330, 'CAKE Earned', { asset: 'CAKE' }).split(' ')
```

Comment:
You don't need to use a translation here , furthermore, splitting like doesn't guarantee to return an acceptable result. What takes 2 words in English, can take 1 or 3 in other langs.

After the tokens reworks, you can get the token symbol with `farm.token` and `farm.quoteToken`. Have a look at the `/state/types` file

Reply:
👍🏻 sure

Reply:
We don't have earned token info in farm object therefore I reverted that code stub